core/ops_builtin.rs

```diff
@@ -36,6 +37,10 @@ pub(crate) fn init_builtins() -> Extension {
       ("op_metrics", op_sync(op_metrics)),
       ("op_void_sync", void_op_sync()),
       ("op_void_async", void_op_async()),
+      // TODO(@AaronO): track IO metrics for builtin streams
```

Comment:
Is there a significant problem with tracking metrics?

Reply:
Not really, there's some debate about how to aggregate/report them but otherwise it's straightforward, but it doesn't fit in the scope of a first-pass IMO